tests/test_mol.py

```diff
@@ -448,3 +454,48 @@ def test_add_remove_hs():
 
     mol3 = dm.remove_hs(mol)
     assert dm.to_smiles(mol3) == smiles
+
+
+def test_unique_id():
```

Comment:
minor, but it's not obvious at a glance that you're testing for different tautomer forms of the same molecule, as per the definition of the function, but maybe you are!

Reply:
`test_unique_id` tests the `dm.unique_id()` function. It's not really related to tautomerism.

But you're right as it is the original purpose of the PR :-)

The two last molecules in `test_unique_id` are actually two tautomers of the same molecule (other mols are just a small list coming from freesolv).

I will add a comment stipulating the above to `test_unique_id`.

Ideally, I would like `unique_id` to always return the same id for the same input even in future datamol/rdkit versions. The test is also a good sanity check for this.